packages/web-components/src/components/va-minimal-header/va-minimal-header.scss

```diff
@@ -0,0 +1,35 @@
+@forward 'settings';
+
+.va-flex{
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 62.5em;
```

Comment:
Are we still trying to just use `px` instead of `rem` because of the v1/v3 root font size clash? If so, then you will probably need to convert all the em/rem in the CSS to px.

Reply:
Converted to px